draft-ietf-quic-recovery.md

```diff
@@ -1499,15 +1477,14 @@ Invoked when an ACK frame with an ECN section is received from the peer.
 Invoked from DetectLostPackets when packets are deemed lost.
 
 ~~~
-   InPersistentCongestion(largest_lost_packet):
+   InPersistentCongestion(lost_packets):
      pto = smoothed_rtt + max(4 * rttvar, kGranularity) +
        max_ack_delay
      congestion_period = pto * kPersistentCongestionThreshold
      // Determine if all packets in the time period before the
      // newest lost packet, including the edges, are marked
```

Comment:
```suggestion
     // largest newly lost packet, including the edges, are marked
```

Reply:
I'm not sure largest makes this less clearer, at least to me.